docs/Examples/Chunked/Main.purs

```diff
@@ -17,7 +17,7 @@ runScript script = EffectClass.liftEffect $ ChildProcess.stdout <$>
 -- | Say 'hello world!' in chunks when run
 sayHello :: HTTPure.Request -> HTTPure.ResponseM
 sayHello =
-  const $ runScript "echo -n 'hello '; sleep 1; echo -n 'world!'" >>= HTTPure.ok
+  const $ runScript "/bin/echo -n 'hello '; sleep 1; /bin/echo -n 'world!'" >>= HTTPure.ok
```

Comment:
This is nonportable. Echo does not necessarily live in /bin. It's pretty surprising this would cause an issue as echo should always be in your path on any system.